src/amoc_lager_logger_formatter.erl

```diff
@@ -1,9 +1,14 @@
+%% This module is copied from: https://github.com/erlang-lager/lager/blob/adt/lager_use_logger-option/src/lager_logger_formatter.erl
+%% with amendments. Original comments left intact.
+%% Fixed timestamp formatting, added catch all clause for report_cb.
```

Comment:
Can we contribute back the timestamp fix?

Reply:
Or all the changes? I think we'll need this formatter in other places like `amoc-arsenal-xmpp` and possibly `MongooseIM`. It's better to use one file instead of coping it to every repo.

Reply:
The whole thing is quite controversial. Maybe I focused on the wrong thing which is the seamless combination of lager and logger.

Unfortunately the integration of the two by the lager author Andrew Thompson is left not quite finished. It seems to work, but it is in a feature branch of lager, not promoted to a release (as the author announced at his presentation this spring in SF - https://codesync.global/media/erlang-logging-for-the-21st-otp-andrew-thompson/).
Given the state of affairs I wouldn't recommend including lager_logger integration in Mongoose yet, let's wait for the next release of Lager).

Should we include it in Amoc at all? Well perhaps as a way of testing it, for example performance and be prepared for the final version.

Reply:
Based on this issue https://github.com/erlang-lager/lager/issues/519 I think the author may need some external help to get this integration moving. I think we can try contributing the fixes and see if they get accepted.